client/src/index.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import { Route, BrowserRouter as Router } from 'react-router-dom'
+import Profile from './Components/Pages/Profile';
```

Comment:
Shouldn't we just have Pages at the root level? Seem strange to nest them inside components.